cfpb/consumerfinance.gov
# Email signups bugfix

Closes https://github.local/CFGOV/platform/issues/4459

Testing:
 - Pull and build
 - visit localhost:8000/consumer-tools/debt-collection/
 - No console error should be reported
 - Email signups should still work.

This bug happened because the code didn't anticipate that an email-signup with no form would coexist with one that had a form. Such is the case on the debt collection tool page.
